packages/react-icons/src/utils/createSVGIcon.js

```diff
@@ -0,0 +1,23 @@
+import React, { forwardRef, memo } from 'react';
+import SVGIcon from '../SVGIcon';
+
+const createSVGIcon = (svgIcon, options) => {
```

Comment:
I like this utility. I was planning on adding TS typings for this package since I have been modifying the typings for `@tonic-ui/react` lately and this makes the task way easier.